shell/Dockerfile

```diff
@@ -27,7 +27,7 @@ RUN apt-get install -y --allow-unauthenticated -q \
     python3-sawtooth-rest-api
 
 RUN curl -sL https://deb.nodesource.com/setup_6.x | bash - \
-    && apt-get install -y nodejs
+    && apt-get install -y nodejs npm nodejs-legacy
```

Comment:
I tried this change on Ubuntu 16.04.6 (Xenial) and I get this error (below). Perhaps your problem is specific to another Ubuntu version or another distribution?

```
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 nodejs : Conflicts: nodejs-legacy
          Conflicts: npm
E: Unable to correct problems, you have held broken packages.
ERROR: Service 'simple-supply-shell' failed to build: The command '/bin/sh -c curl -sL https://deb.nodesource.com/setup_6.x | bash -     && apt-get install -y nodejs npm nodejs-legacy' returned a non-zero code: 100
```

Reply:
You are right; it's a version specific problem. Mine ubuntu is 14.04 .